### PR TITLE
Update reference for bug bounty program for node core

### PR DESCRIPTION
**Pull request by lirantal**
Follow-up from #49 

Updates necessary:
- [x] Update Security WG README for clearer communication
- [x] Add a similar reference/link on the HackerOne platform's landing page for the `/nodejs-ecosystem` to refer researches to `/nodejs` for core Node.js issues
- [x] PR to the Node.js website to add similar reference as in our README 

Above makes sense? any other?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ security activities for Node.js core but relies on the Node.js Security Working
 Group to recommend and help maintain policies and procedures for that
 management.
 
+### Node.js Bug Bounty Program
+
+The Node.js project engages in an official bug bounty program for security researches and responsible public disclosures.
```
Comment (drifkin) [inline README.md]: researches -> researchers

Comment (mhdawson) [inline README.md]: researches -> researchers

File: README.md [hunk 2]
```diff
@@ -52,6 +52,13 @@ security activities for Node.js core but relies on the Node.js Security Working
...
+
+The program is managed through the HackerOne platform at [https://hackerone.com/nodejs](https://hackerone.com/nodejs) with further details.
```
Comment (mhdawson) [inline README.md]: Maybe ...platform. You can find more details on this page: [https...

Comment (drifkin) [inline README.md]: if the URL is the same as the text, how about just putting it in angle brackets like:

```
the HackerOne platform at <https://hackerone.com/nodejs>
```

File: README.md (hunk 1 above)
Comment (drifkin) [reply README.md]: Whoops, @mhdawson beat me to it :)

Comment (lirantal) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (lirantal) [reply README.md]: do you mean this?
```
The program is managed through the HackerOne platform.
You can find more details on this page: https://hackerone.com/nodejs
```

File: README.md (hunk 1 above)
Comment (lirantal) [reply README.md]: :-)

File: README.md (hunk 2 above)
Comment (lirantal) [reply README.md]: In other places we just use the URL so I'll change it to plain text

**Comment by lirantal** [comment]
@mhdawson can you check my reply to your suggestion, I'm not sure I understood it.
https://github.com/nodejs/security-wg/pull/169#discussion_r175934466

**Comment by mhdawson** [comment]
We should wait to land this until we clarify if/when core will start using Hacker one for reporting.

**Comment by reedloden** [comment]
Also submitted https://github.com/nodejs/nodejs.org/pull/1627

**Comment by lirantal** [comment]
Yes!

Awesome and thanks @reedloden for PRing to the website 👏

**Comment by davisjam** [comment]
Anyone want to send me a check for my path REDOS disclosures? ;-)

**Comment by reedloden** [comment]
@davisjam The bug bounty program is only for security issues that affect Node.js core itself, not any third-party node modules.

**Comment by reedloden** [comment]
@davisjam ah, neat. My apologies. I know the NSP had reported a bunch of ReDoS issues in third-party modules, so thought it was related to something similar.

In any case, the program is not retroactive, so any already reported and fixed issues are not eligible. However, new issues that meet the minimum bar will be eligible. :-)

**Pull request merged**